QuickBytes/src/main/java/com/quickbytes/repository/ItemRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.quickbytes.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.quickbytes.model.Item;
+
+
+
+public interface ItemRepository extends JpaRepository<Item,Long> {
+	
+	@Query("select i from Item i where i.vendor.vendorId = ?1")
+	List<Item> findByAccountId(Long vid);
```

Comment:
Maybe change to findByVendorId to make sure we know what type of account we're referring to?

Reply:
read, will change (for all)